src/lib/apiClient.ts

```
@@ -281,65 +373,425 @@ export interface PriceHistoryResponse {
+      const errorMessage =
+        data?.error?.message ||
+        data?.error ||
+        `Failed to submit borrow: ${response.statusText}`;
+      throw new Error(errorMessage);
+    }
+
+    return data as LiquidiumSubmitBorrowResponse;
+  } catch (error) {
+    throw error;
+  }
+};
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)